workspace-manager-clienttests/render-config.sh

```diff
@@ -1,9 +1,23 @@
 #!/bin/bash
 
-VAULT_TOKEN=${1:-$(cat "$HOME"/.vault-token)}
-WM_APP_SERVICE_ACCOUNT_VAULT_PATH=secret/dsde/terra/kernel/dev/dev/workspace/app-sa
+ENV=${1:-local}
+VAULT_TOKEN=${2:-$(cat "$HOME"/.vault-token)}
+
+if [ -z "${ENV}" ]; then
```

Comment:
nit: naming it `$ENV` might be confusing, if this is talking about the terra  env only.  I got confused w/r/t the `env` command just now.

Reply:
I can rename it WORKSPACE_MANAGER_ENV if that is clearer